presto-main/src/main/java/io/prestosql/execution/CallTask.java

```diff
@@ -94,6 +94,13 @@ public String getName()
             throw semanticException(INVALID_ARGUMENTS, call, "Named and positional arguments cannot be mixed");
         }
 
+        int optionalArgumentsCount = (int) procedure.getArguments()
```

Comment:
Formatting nit: we put stream on the same line as the expression
```java
 int optionalArgumentsCount = (int) procedure.getArguments().stream()
        .filter(...)

Reply:
Use `toIntExact` instead of cast to int, as that makes the intention clear and avoids a warning. (it won't ever be larger, but if it was, we wouldn't want to do a lossy conversion)